### ISSUE TITLE
startOfMonth(format(new Date(), "MMM yyyy")) invalid date in iOS and macOS

### ISSUE THREAD

**Issue by spider9375** [date-fns/date-fns#3910]
startOfMonth returns invalid date when the date when in format "MMMM yyyy" in macOS and iOS Safari.
Works in Chrome...

**Comment by Frost7994** [comment]
Did you figure this out? @spider9375

**Comment by spider9375** [comment]
Yes. What I did was first `parse` the initial date and also pass 3rd argument which is `new Date()` - this way the `parse` function was able to resolve the date correctly no matter the device.

In the end it looks like that

`
import {parse, startOfMonth} from "date-fns";

private todaysMonth = format(new Date(), 'MMMM yyyy');
private foo = startOfMonth(parse(this.todaysMonth, 'MMMM yyyy', new Date()))
`
